system/Test/Mock/MockCookieConfig.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Test\Mock;
+
+use Config\Cookie;
+
+class MockCookieConfig extends Cookie
```

Comment:
Do not add useless mocks. Mocks can be easily made natively with PHPUnit's mocking system.